Realtime/Socket.cs

```diff
@@ -104,38 +106,34 @@ private async void DisposeConnection()
         /// </summary>
         public async void Connect()
```

Comment:
Since this is a bunch of breaking changes, it could be worthwhile to change this to `async Task` right now too.  `async void` should generally be avoided where possible.

Reply:
Good call!